staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -129,22 +130,36 @@ func (s *SpdyRoundTripper) dial(req *http.Request) (net.Conn, error) {
 		return s.dialWithoutProxy(req.Context(), req.URL)
 	}
 
+	switch proxyURL.Scheme {
+	case "socks5":
+		return s.dialWithSocks5Proxy(req, proxyURL)
+	case "https", "http":
```

Comment:
from the docs for ProxyFromEnvironment:

> The environment values may be either a complete URL or a "host[:port]", in which case the "http" scheme is assumed.

should "" be included in this case as well?